paths/unique_identifiers.yaml

```diff
@@ -5,6 +5,15 @@ get:
   description: |
     Get uuid for given model currently supported models are Tag, StaticPage, Producttest, PrivateGroup, Category, Question, Feedback, Article, Faq, Tutorial, ProductReview, ProducttestApplication, WidgetContainer
   parameters:
+    - name: user
+      in: body
```

Comment:
parameters in the body are marked up differently in the openapi version 2
do you mean query?

Reply:
nope in body.

Reply:
but then it shouldn't be get. GET is not standardized to parse GET bodies, should such exist

Reply:
ah I overlooked it 
it is actually query params, 
http://localhost:3000/api/2.1/unique_identifiers?tag[]=enrollee&question[]=asdasdasdadasd&static_page[]=dasdas&user[]=testadmin